### PR TITLE
feat: change new return layout

### PR DESCRIPTION
**Pull request by luisaha**
## Why:

Closes: #96647 in Azure

## Checklist before requesting a review

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue [hunk 1]
```diff
@@ -104,14 +106,63 @@
...
+                :class="index === orderGetters.getItemOrderProperties(orderItem).length - 1 ? 'rounded-b-md' : ''"
+              >
+                <div class="flex justify-between">
+                  <p class="text-sm px-2">{{ orderGetters.getItemOrderPropertyName(property) }}</p>
+                  <p class="text-sm px-2 text-gray-300">{{ orderGetters.getItemOrderPropertyValue(property) }}</p>
+                </div>
+              </h3>
+            </div>
+            <h3
+              v-if="orderGetters.getOrderAttributes(props.orderItem).length === 0"
```
Comment (maxiroellplenty) [inline apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue]: this should only be displayed if there are no properties and attributes available

File: apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue [hunk 2]
```diff
@@ -104,14 +106,63 @@
             </SfSelect>
           </label>
         </div>
+
+        <div class="my-2 w-full md:self-end">
+          <button
```
Comment (maxiroellplenty) [inline apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue]: Why don't we use this component? https://docs.storefrontui.io/v2/vue/components/accordionitem.html#accordion-group

File: apps/web/pages/my-account/new-return/[id].vue [hunk 3]
```diff
@@ -0,0 +1,119 @@
+<template>
+  <UiDivider class="col-span-3 -mx-4 !w-auto md:mx-0" />
+  <h2
+    v-if="!confirmation"
+    class="hidden md:block col-span-3 typography-headline-4 font-bold mx-4 capitalize"
+    data-testid="account-new-return-heading"
+  >
+    {{ t('returns.return') }}
+  </h2>
+  <div v-if="!confirmation" class="col-span-3 mt-8 !w-auto">
```
Comment (maxiroellplenty) [inline apps/web/pages/my-account/new-return/[id].vue]: Bildschirmfoto 2024-02-29 um 16 24 49

Should have the same margin as the heading

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

File: apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue (hunk 1 above)
Comment (maxiroellplenty) [reply apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue]: Bildschirmfoto 2024-02-29 um 16 10 57

File: apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue (hunk 2 above)
Comment (luisaha) [reply apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue]: I think that's a question, which needs to be answered by Tim. We just followed the look of the mockups.

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**